pkg/routeagent/controllers/route/iptables.go

```diff
@@ -17,35 +17,46 @@ func (r *Controller) createIPTableChains() error {
 
 	klog.V(4).Infof("Install/ensure %s chain exists", SmPostRoutingChain)
 	if err = r.createChainIfNotExists(ipt, "nat", SmPostRoutingChain); err != nil {
-		return fmt.Errorf("Unable to create %s chain in iptables: %v", SmPostRoutingChain, err)
+		return fmt.Errorf("unable to create %s chain in iptables: %v", SmPostRoutingChain, err)
 	}
 
 	klog.V(4).Infof("Insert %s rule that has rules for inter-cluster traffic", SmPostRoutingChain)
 	forwardToSubPostroutingRuleSpec := []string{"-j", SmPostRoutingChain}
 	if err = r.prependUnique(ipt, "nat", "POSTROUTING", forwardToSubPostroutingRuleSpec); err != nil {
-		klog.Errorf("Unable to insert iptable rule in NAT table, POSTROUTING chain: %v", err)
+		klog.Errorf("unable to insert iptable rule in NAT table, POSTROUTING chain: %v", err)
```

Comment:
why klog.Error versus return in other cases (L20,L31,L36 ..) may be we should also bubble up the error here for consistency

Reply:
Oops, I missed it, thanks for pointing out. Will update the PR.